docs/server/requirements.rst

```diff
@@ -0,0 +1,98 @@
+.. _requirements:
+
+Pootle requirements
+===================
+
+.. _requirements#hardware:
+
+Hardware Requirements
+---------------------
+
+Your Pootle installation will need to be flexible enough to handle the
+translation load. The recommended hardware depends highly on the performance you
+expect, the number of users you want to support, and the number and size of the
+files you want to host.
+
+Whatever hardware you have, you will still benefit from performance improvements
+if you can :doc:`optimize your system <optimization>`.
+
+Your disk space should always be enough to store your files and your Pootle
+database, with some extra space available.
+
+
+.. _requirements#system:
+
+System Requirements
+-------------------
+
+To run Pootle you need a computer running:
+
+- Linux
+- Mac OS X
+
+Or, any other Unix-like system.
+
+.. note:: Pootle will not run on Windows since it uses RQ, whose workers cannot
+   run on `Windows <http://python-rq.org/docs/>`_.
+
+   Some developers do develop on Windows so these problems can be worked around
+   for some of the development tasks.
+
+   Pootle should be able to run on any system that implements ``fork()``.
+
+
+.. _requirements#python:
+
+Python version
+--------------
+
+**Python 2.7 is required**. 2.6 won't work, and 3.x is not supported.
+
+
+.. _requirements#packages:
+
+Installing system packages
+--------------------------
+
+You will need a C compiler and  the development libraries for Python and XML to
+be available on your system before you will be able to install your virtual
+environment. You will also need pip.
+
+Eg. on a Debian-based system:
+
+.. code-block:: bash
+
+  $ sudo apt-get install \
+      build-essential libxml2-dev libxslt-dev python-dev python-pip
```

Comment:
Please put this in a single line. In other words, get rid of that `\` so people can easily copy this.